.github/workflows/cspellcdict.txt

```diff
@@ -3162,3 +3182,22 @@ Gmail
 vmf
 mdl
 html
+VANTS
+Dont
+PHQ
+PCL
+vis
```

Comment:
Should this be **Viz** and **viz** ?

Reply:
There is a html place holder called (#DontLink) that keeps setting off the spell checker.  So I added "Dont" to the list. Not sure about "vis." and where it may have, or not have been used in the documentation.